haproxy/assets/configuration/spec.yaml

```
@@ -12,7 +12,8 @@ files:
         example: false
         type: boolean
       description: |
-        Whether or not to preview the new version of the check supporting only HAProxy versions >=2.
+        Whether or not to preview the new version of the check supporting only HAProxy versions >=2,
+        or using the HAProxy exporter.
```

Can we rephrase to drop `whether or not`, at first read it's confusing what this enables. Maybe something like:

```suggestion
        Enable to preview the new version of the check which supports HAProxy version 2 or higher or environments using the HAProxy exporter.
```

---

+1 to this change here, and to the conf.yaml.example below